Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -567,6 +566,28 @@ static void addAttributesForFontPalettes(CFMutableDictionaryRef attributes, cons
     }
 }
 
+static std::optional<bool>& overrideEnhanceTextLegibilityStorage()
```

Comment:
Can you call this just `overrideEnhanceTextLegibility()` and call the one below `setOverrideEnhanceTextLegibility(bool)` ?